lib/CybsNameValuePairClient.php

```diff
@@ -11,9 +11,11 @@
 class CybsNameValuePairClient extends CybsClient
 {
 
-    function __construct($options=array())
+    function __construct($options=array(), $properties=array())
```

Comment:
That change would break any downstream code using `$options` as the first parameter. I don't think that's quite possible at this point.